Jan 6 Cloud release notes (#273)
astronomer/docs: cloud/release-notes.md

**Latest CLI Version**: 1.0.4 ([Release notes](cli-release-notes.md))

## January 6, 2021

> ### Logging Improvements

Reviewer: ```suggestion
### Improvements to "Scheduler Logs" in the Astronomer UI
```

> The **Logs** tab has been updated to make it easier to read and organize. Specifically:

Reviewer: ```suggestion
The **Scheduler Logs** tab has been updated to make it easier to read and organize. Specifically:
```

> - You can now filter logs by type (`DEBUG`, `INFO`, `WARN`, and `ERROR`).
> - The console now shows logs for the past 24 hours.
> - The console can now show a maximum of 500 logs.

Reviewer: ```suggestion
- You can now filter logs by type (`DEBUG`, `INFO`, `WARN`, and `ERROR`).
- The page now shows logs for the past 24 hours, instead of the past 30 minutes.
- The page now shows a maximum of 500 logs, instead of a lower maximum.
```

> - When looking at a Deployment's logs, you can jump back to the Deployment using the **Deployment Details** button.

Reviewer: ```suggestion
- When looking at a Deployment's logs, you can navigate back to the Deployment using the **Deployment Details** button.
```

![Logs page in the UI](/img/release-notes/log-improvements.png)

### Additional Improvements

> - Removed inaccurate _Kubernetes Version_ column from the **Clusters** table.

Reviewer: ```suggestion
- Removed _Kubernetes Version_ column from the **Clusters** table. This value was previously inaccurate, and is not needed. The Kubernetes version of any particular Astronomer Cluster is set and modified exclusively by Astronomer as part of our managed service.
```

![Logging direct access button](/img/release-notes/logs-button.png)

> For more information on how to view logs, read [Deployment Logs](deployment-logs.md).

Reviewer: ```suggestion
For more information on how to view logs, read [Scheduler Logs](scheduler-logs.md).
```
